### ISSUE TITLE
apoc.path.expand result Expected to be a path

### ISSUE THREAD

**Issue by stdob** [neo4j-contrib/neo4j-apoc-procedures#57]
This query:

```
MERGE (A1:Crx {id:1})-[:inx]->(A2:Crx {id:2})-[:inx]->(A3:Crx {id:3})
WITH A1
CALL apoc.path.expand( A1, 'inx>', '+Crx', 1, 3) YIELD path AS pathway
RETURN A1.id, NODES(pathway)
```

Gives an error message:

> Expected pathway to be a path.
> Neo.ClientError.Statement.SyntaxError

This query operates without errors:

```
MERGE (A1:Crx {id:1})-[:inx]->(A2:Crx {id:2})-[:inx]->(A3:Crx {id:3})
WITH A1
CALL apoc.path.expand( A1, 'inx>', '+Crx', 1, 3) YIELD path AS pathway
RETURN A1.id, pathway
```

**Comment by jexp** [comment]
Fixed in 3.0.3 please upgrade and confirm.

**Comment by stdob** [comment]
Okay, I tested with version 3.0.3. Everything is fine.

But there is another problem: build failure with `mvn clean install`. It does not pass the tests:

> Tests in error:
>   shouldGetPageRankExpectedResult(apoc.algo.PageRankTest): Failed to invoke procedure `apoc.algo.pageRankWithConfig`: Caused by: java.lang.RuntimeException: Error encountered while calculating page rank
>   shouldHandleEmptyNodeSet(apoc.algo.PageRankTest): Failed to invoke procedure `apoc.algo.pageRank`: Caused by: java.lang.RuntimeException: Error encountered while calculating page rank
>   shouldGetPageRankExpectedResultWithTypes(apoc.algo.PageRankTest): Failed to invoke procedure `apoc.algo.pageRankWithConfig`: Caused by: java.lang.RuntimeException: Error encountered while calculating page rank
>   shouldGetPageRankArrayStorageSPI(apoc.algo.pagerank.PageRankAlgoTest): Invalid input 'u': expected 'e/E' (line 1, column 9 (offset: 8))
>   shouldGetPageRankArrayStorageSPIWithTypes(apoc.algo.pagerank.PageRankAlgoTest): Invalid input 'u': expected 'e/E' (line 1, column 9 (offset: 8))

So I do build skipping tests: ` mvn -Dmaven.test.skip=true install`
